### PR TITLE
added missing query registration for MITS emails

### PR DESCRIPTION
**Pull request by EliAndrewC**
I forgot to register the ``MITSTeam`` class with the ``AutomatedEmail.queries`` dict.  This does that, so that those emails can actually be sent.

### CODE REVIEW

File: mits/automated_emails.py [hunk 1]
```diff
@@ -6,11 +6,13 @@ def __init__(self, *args, **kwargs):
...
         AutomatedEmail.__init__(self, MITSTeam, *args, **kwargs)
 
+AutomatedEmail.queries[MITSTeam] = lambda session: session.mits_teams()
+
 # We wait an hour before sending out this email because the most common case
 # of someone registering their team is that they'll immediately fill out the
 # entire application, so there's no reason to send them an email showing their
 # currently completion percentage when that info will probably be out of date
-# by the time they read it.  By waiting an hour, we 
+# by the time they read it.  By waiting an hour, we ensure this doesn't happen.
```
Comment (EliAndrewC) [inline mits/automated_emails.py:15]: Unrelated fix; I noticed that the end of this sentence had gotten cut off somehow.  Or possibly I just got distracted when writing it in the first place in the middle of

**Review by kitsuta — APPROVED** [review]
Huh! I didn't realize you had to register automated emails for new plugins. Oops!
